cmake/Dependencies.cmake

```diff
@@ -216,6 +216,18 @@ if(USE_FFTW OR NOT MKL_FOUND)
   endif()
 endif()
 
+# --- [ PocketFFT
+set(AT_POCKETFFT_ENABLED 0)
+if(NOT MKL_FOUND)
+  find_path(POCKETFFT_INCLUDE_DIR NAMES pocketfft_hdronly.h
+            PATHS /usr/local/include
+            PATHS $ENV{POCKETFFT_HOME}
+           )
```

Comment:
Curious, why not just include in `third_party`?

Reply:
Because it is not hosted on github, which might be hard to sync up if used frequently.
But might create a mirror on github and add to 3rd party later on